src/ripple/app/main/Application.cpp

```diff
@@ -78,7 +78,9 @@
 namespace ripple {
 
 // 204/256 about 80%
-static int const MAJORITY_FRACTION(204);
+static int const MAJORITY_FRACTION_OLD(204);
+// 8/10 = 80%
+static int const MAJORITY_FRACTION(8);
```

Comment:
These values, along with the default value for the majority time should probably migrate into `SystemParametersInfo.h`:

```
/** The minimum amount of support an amendment should have.

    @note This value is used by legacy code and will become obsolete
          once the fix3396 amendment activates.
*/
constexpr int const AmendmentMajorityFractionOld = 204;

/** The minimum amount of support an amendment must have */
constexpr int const AmendmentMajorityPercent = 80;

/** The minimum amount of time an amendment must hold a majority */
constexpr std::chrono::seconds const AmendmentMajorityTime = std::chrono::weeks{2};

```

Reply:
Done, except for `amendmentMajorityPercent`, which is currently not used. Added instead `amendmentMajorityFraction`.